perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -525,6 +525,27 @@ sub tokenised_url {
     return "/M/". $token->token;
 }
 
+=head2 view_url
+
+Return a url for this problem report that will always show it
+(even if e.g. a private report) but does not log the user in.
+
+=cut
+
+sub view_url {
```

Comment:
Calling this twice in the HTML and plaintext versions of the email creates two tokens. Not a big deal but seems like it could be avoided.